### DISCUSSION TITLE
1.4.1rc1

### DISCUSSION THREAD

**Discussion by eschutho** [apache/superset#18187] [General]
Hello Superset Community,

Please add any bug reports here for 1.4.1rc1

The release candidate:
https://dist.apache.org/repos/dist/dev/superset/1.4.1rc1/

Git tag for the release:
https://github.com/apache/superset/tree/1.4.1rc1

The Change Log for the release:
https://github.com/apache/superset/blob/1.4.1rc1/CHANGELOG.md

The Updating instructions for the release:
https://github.com/apache/superset/blob/1.4.1rc1/UPDATING.md

public keys are available at:

https://www.apache.org/dist/superset/KEYS

**Comment by iercan** [comment]
I'd appreciate If you can include #18198

**Reply by fritzb** [reply]
This is an important fix. We can't use 1.4 on a large dashboard without this fix.

**Reply by eschutho** [reply]
It looks like the fix for #18198 is https://github.com/apache/superset/pull/18181. Although we still don't know why this specifically started happening for you on 1.4.0, I'm given to bring this cherry in for 1.4.2 if the vote for 1.4.1 passes, and will pull in for 1.4.1 if another rc goes out.

**Reply by villebro** [reply]
This is going to be very difficult to include in the 1.4 branch, as it requires many prerequisite PRs from master branch that most likely won't apply cleanly to this branch. I believe we're missing another cherry that's needed to sanitize the form data more. Let's continue the discussion on #18198 and try to get to the bottom of what's causing the overflow.

**Comment by sowo** [comment]
I'd appreciate if commit 6e5392d (fix(textarea-control): ace editor input exception) could be cherry picked into 1.4.1

**Comment by iercan** [comment]
We detected another important bug too. Please include  #18220  too

**Reply by rumbin** [reply]
@iercan I was about to mention this bug here, too.
This is a real show stopper for us, as this bug will be easily stumbled upon by our freshly trained users who often start their journey by cloning an existing dashboard.

**Comment by eschutho** [comment]
Thank you everyone for flagging those important fixes. If the current vote for 1.4.1 passes, I will bring all of these in for 1.4.2. If there's another rc for 1.4.1, then I'll bring these fixes in for this version.

**Comment by gnought** [comment]
I'm trying to track down how many commits we should add before switching the monorepo, so we could get closer to a full working copy and easier to migrate. The following commits seems we could consider. 
#16999
#16974
#17008
#17001
#17016
#17024
#16959
#16993
#17045
#14577
#17039
#17102
#16154, seems it should merge before #17037
#17055
#17147
#17208
#17130
#17174
#17155
#17134
#17168
#17166
#17177
#17046
#17112
#17196
#17157
#17215

**Reply by eschutho** [reply]
@gnought the monorepo will be a part of the next minor release (1.5) and there shouldn't be anything extra to do to migrate to that version. I'm not sure of your specific setup though if you want to elaborate more on how you are planning to migrate.
